themes/docsy/assets/scss/_custom.scss

```diff
@@ -903,6 +903,69 @@ body.tv-episode {
   margin-right: 10px;
 }
 
+.checklist-item {
+  padding: 2rem 2rem 1rem;
+  margin: 1.5rem 0;
+  display: flex;
+  border-left: 3px solid var(--MAIN-ANCHOR-color);
+}
+
+.checklist-item > input[type="checkbox"] {
+  margin-right: 1em;
+  margin-top: 5px;
+}
+
+.checklist-item h4 {
+  margin: 0;
+}
+
+table {
```

Comment:
We can address this later but we might want to be careful about setting global table styling. There is a `table` class I had been using and wrapping Markdown tables with `{{< table "table" >}}` as in https://github.com/vmware-tanzu/tanzu-dev-portal/blob/main/content/guides/microservices/distributed-tracing.md

Reply:
Oops missed this when searching, have altered to use your style and removed mine.